docs/source/examples/demo_transformer_model_train_save_upload_to_openSearch.ipynb

```diff
@@ -1095,8 +1212,8 @@
    "source": [
     "# default to download model id, \"sentence-transformers/msmarco-distilbert-base-tas-b\" from hugging face \n",
     "# and output a model in a zip file containing model.pt file and tokenizers.json file. \n",
-    "model = SentenceTransformerModel()\n",
-    "model.save_as_pt(sentences = ['today is sunny'])"
+    "pre_trained_model = SentenceTransformerModel(folder_path = '/home/ec2-user/SageMaker/opensearch-py-ml-mingshl/export_huggingface/', overwrite = True)\n",
```

Comment:
Not a big problem, but suggest to not add personal information on the public doc , I see your name in `opensearch-py-ml-mingshl`

Reply:
will rename the folder name to `opensearch-py-ml-test`